go.mod

```diff
@@ -5,10 +5,10 @@ go 1.21
 toolchain go1.21.1
 
 require (
-	github.com/docker/buildx v0.12.2-0.20240220084849-89154c7d3303 // indirect
-	github.com/docker/cli v25.0.4-0.20240221083216-f67e569a8fb9+incompatible // indirect
+	github.com/docker/buildx v0.13.0 // indirect
+	github.com/docker/cli v26.0.0-rc1+incompatible // indirect
```

Comment:
Hum it updates peer deps as well :flushed: related to our go mod upstream: 
* https://github.com/docker/buildx/blob/v0.13.0/go.mod#L19
* https://github.com/docker/buildx/blob/v0.13.0/go.mod#L28

Don't think we want this. I think we always want to be on master for buildkit and current stable for docker.

Is `hugo mod` using the same semantic as `go mod`? If not, do they have a flag to not update indirect deps? If same semantic we should probably only use `replace` directives.

Reply:
as discussed offline — using `go mod edit` seems to work better 🙄